### PR TITLE
update_grpc_testcase

### PR DESCRIPTION
**Pull request by pbajjuri20**
Tested against the 2.4 and 2.3, it passed successfully.

2.4 --> meshConfig
2.3 --> techPreview

### CODE REVIEW

File: pkg/tests/tasks/security/authorization/ext_auth_test.go [hunk 1]
```diff
@@ -136,7 +136,25 @@ func TestEnvoyExtAuthzGrpcExtensionProvider(t *testing.T) {
 		oc.WaitDeploymentRolloutComplete(t, ns, "ext-authz")
 
 		t.LogStep("Set envoyExtAuthzgRPC extension provider in SMCP")
-		if env.GetSMCPVersion().GreaterThanOrEqual(version.SMCP_2_3) {
+		if env.GetSMCPVersion().GreaterThanOrEqual(version.SMCP_2_4) {
```
Comment (monroe-dylan-m) [inline pkg/tests/tasks/security/authorization/ext_auth_test.go:139]:
```suggestion
		if env.GetSMCPVersion().LessThan(version.SMCP_2_4) {
```
This should be `LessThan` and not `GreaterThanOrEqual`, as it is on line 57.

File: tests/result-20230907153355/v2.5/failures-must-gather/20230907153410-TestEnvoyExtAuthzGrpcExtensionProvider/timestamp
Comment (monroe-dylan-m) [inline tests/result-20230907153355/v2.5/failures-must-gather/20230907153410-TestEnvoyExtAuthzGrpcExtensionProvider/timestamp:1]: Looks like this file was included by mistake.

**Review by monroe-dylan-m — CHANGES_REQUESTED** [review]

File: tests/result-20230907153355/v2.5/failures-must-gather/20230907153410-TestEnvoyExtAuthzGrpcExtensionProvider/timestamp
Comment (pbajjuri20) [reply tests/result-20230907153355/v2.5/failures-must-gather/20230907153410-TestEnvoyExtAuthzGrpcExtensionProvider/timestamp:1]: will remove it

**Review by monroe-dylan-m — APPROVED** [review]
LGTM!

**Pull request merged**